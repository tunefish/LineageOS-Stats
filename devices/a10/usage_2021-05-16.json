{
  "10.3": 1,
  "16.0": 82,
  "17.1": 1031,
  "18.1": 703,
  "none": 610
}