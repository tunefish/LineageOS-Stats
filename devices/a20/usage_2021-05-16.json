{
  "17.1": 2438,
  "18.1": 372,
  "none": 28
}