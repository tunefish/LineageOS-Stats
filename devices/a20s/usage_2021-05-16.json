{
  "15.1": 1,
  "17.1": 2499,
  "none": 1
}