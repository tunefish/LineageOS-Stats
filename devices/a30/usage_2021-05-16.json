{
  "17.1": 40,
  "18.0": 800,
  "18.1": 581,
  "none": 4
}