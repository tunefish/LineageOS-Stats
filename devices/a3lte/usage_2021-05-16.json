{
  "15.1": 808,
  "16.0": 301,
  "17.1": 1073
}