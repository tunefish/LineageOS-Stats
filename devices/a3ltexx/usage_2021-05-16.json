{
  "14.1": 1318
}