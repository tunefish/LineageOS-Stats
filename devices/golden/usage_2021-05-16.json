{
  "13.0": 1426,
  "14.1": 5138,
  "15.1": 6,
  "17.1": 14,
  "none": 1
}